gempython/tools/amc_user_functions_uhal.py

```diff
@@ -172,3 +172,10 @@ def blockL1A(device):
 
 def enableL1A(device):
     writeRegister(device, "GEM_AMC.TTC.CTRL.L1A_ENABLE", 0x1)
+
+def getAMCL1Acount(device):
+    return readRegister(device, "GEM_AMC.TTC.L1A_ID")
+                                                                                          
+def getLinkL1Acount(device,link):
+    return readRegister(device, "GEM_AMC.DAQ.OH%s.COUNTERS.EVN"%(link))
+
```

Comment:
This has been fixed already and this function migrated to optohybrid_user_functions